applications/matter_weather_station/src/zap-generated/CHIPClusters.h

```diff
@@ -27,20 +27,15 @@
 #include <lib/core/CHIPCallback.h>
 #include <lib/support/Span.h>
 
-namespace chip
-{
-namespace Controller
+namespace chip {
```

Comment:
We generally don't clang-format zap-generated code.

Reply:
Ok, I've never done that. We typically use `generate.py` script for that.

Reply:
I tried to repair that, and maybe now it looks better. :)